pubspec.yaml

```diff
@@ -1,5 +1,5 @@
 name: code_builder
-version: 1.0.0-alpha+2
+version: 1.0.0-alpha+3
```

Comment:
I'm not really sure about the standard practice here, but should we bump the version # given a breaking change? Or because this is all still alpha, we're not too worried?